app/scripts/components/common/banner/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React, {useState} from "react";
+import { Banner as USWDSBanner, BannerContent, Icon } from "@trussworks/react-uswds";
+
+const BANNER_KEY = 'dismissedBannerUrl';
+
+function hasExpired(expiryDatetime) {
+  const expiryDate = new Date(expiryDatetime);
+  const currentDate = new Date();
+  return !!(currentDate > expiryDate);
+}
+
+enum BannerType {
+  info = 'info',
+  warning ='warning'
+}
+
+const infoTypeFlag = BannerType.info;
+interface BannerProps {
+  appTitle: string,
+  expiryDate: Date,
+  actionUrl: string,
+  contents: string,
+  type?: BannerType
+}
+
+export default function Banner({appTitle, expiryDate, actionUrl, contents, type = infoTypeFlag }: BannerProps) {
+  
+  const showBanner = localStorage.getItem(BANNER_KEY) !== actionUrl;
+  const [isOpen, setIsOpen] = useState(showBanner && !(hasExpired(expiryDate)));
+
+  function onClose () {
+    localStorage.setItem(
+      BANNER_KEY,
+      actionUrl
+    );
+    setIsOpen(false);
+  }
+
+  return (
+    <div>
+      {isOpen && 
+        (<div className='position-relative'>
+          <USWDSBanner aria-label={appTitle} className={type !== infoTypeFlag? 'bg-secondary-lighter': ''}>
```

Comment:
Floagging that this component uses uswds default color theme now: https://designsystem.digital.gov/design-tokens/color/theme-tokens/ 

I don't want to drag this PR with another configuration setup especially considering there hasn't been a request for a different type of banner. (But then it might be better to just get rid of this special styling for now?)

Reply:
Until needed I'm in support of the default theme 👍